File: MangaST_1_11/src/main/java/com/example/mangast/user/UserResponse.java
package com.example.mangast.user;

import com.example.mangast.user.role.Role;
import lombok.*;

import java.time.LocalDate;


@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UserResponse {
    private Integer id;

    private String email;

    private String password;

    private String nickname;

    private Role role;

    private String userPageId;

    private boolean isBanned;

    private boolean isActivated;

    private String firstname;

    private String lastname;

    private String aboutYou;

    private LocalDate birthday;

    private byte[] userCover;




}

File: MangaST_1_11/FrontMangaST/src/app/services/models/user-response.ts
/* tslint:disable */
/* eslint-disable */
export interface UserResponse {
  aboutYou?: string;
  activated?: boolean;
  banned?: boolean;
  birthday?: string;
  email?: string;
  firstname?: string;
  id?: number;
  lastname?: string;
  nickname?: string;
  password?: string;
  role?: 'TEST' | 'USER' | 'ADMIN' | 'MODER';
  userCover?: Array<string>;
  userPageId?: string;
}
